versions.tf

```
@@ -15,15 +15,15 @@
  */
 
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 0.13.0, < 15.0"
```

This is a breaking change and same question as above w.r.t versions.
```suggestion
  required_version = ">= 0.13.0, < 15.0"
```

---

Same as above note.